CSDiscordService/Eval/ScriptExecutionContext.cs

```diff
@@ -44,7 +44,9 @@ public class ScriptExecutionContext
                 "System.Text.Json",
                 "CSDiscordService.Eval",
                 "AngouriMath",
-                "AngouriMath.Extensions"
+                "AngouriMath.Extensions",
+                "HonkSharp.Fluency",
+                "HonkSharp.Functional"
```

Comment:
It's a `using` from AngouriMath's dependency, so it doesn't add anything to repl. Hope it will work XD